pkg/test/env/environment.go

```diff
@@ -91,6 +91,10 @@ func NewEnvironment(t *testing.T, options ...Option) (*Environment, error) {
 		return env, fmt.Errorf("can not create container runner: %w", err)
 	}
 
+	if err = env.runner.PullContainerImages(skeletons); err != nil {
```

Comment:
Does it make sense to integrate this call into `RunContainers`?  
Would it make a little bit more convenient maybe.

Reply:
yeah i thought about that too. Will adjust that, because it's unlikely that someone would want to only use the container runner without the test system around it

Reply:
adjusted that